This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: brodieG <brodieG@users.noreply.github.com>
Date: Mon, 20 Jun 2022 23:44:02 +0000
Subject: [PATCH 1/1] Always link as static

Based on the previous patch by Timothy Gu, updated by
Luke Potgieter <fried.roadkill+ght@gmail.com>, then
adjusted to apply on 3.9.1.

diff --git a/tools/geos-config.in b/tools/geos-config.in
index 1111111..2222222 100644
--- a/tools/geos-config.in
+++ b/tools/geos-config.in
@@ -46,22 +46,11 @@ while test $# -gt 0; do
     --version)
       echo @VERSION@
       ;;
-    --libs)
-      # TODO: make an alias for --clibs
-      # see http://trac.osgeo.org/geos/ticket/497
-      echo -L${libdir} -lgeos-@VERSION_RELEASE@
+    --clibs|--static-clibs)
+      echo -L${libdir} -lgeos_c -lgeos -lm -lstdc++
       ;;
-    --clibs)
-      echo -L${libdir} -lgeos_c
-      ;;
-    --cclibs)
-      echo -L${libdir} -lgeos
-      ;;
-    --static-clibs)
-      echo -L${libdir} -lgeos_c -lgeos -lm
-      ;;
-    --static-cclibs)
-      echo -L${libdir} -lgeos -lm
+    --libs|--cclibs|--static-cclibs)
+      echo -L${libdir} -lgeos -lm -lstdc++
       ;;
     --cflags)
       echo -I${prefix}/include
